### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Mazurok001**
- [DEMO LINK](https://Mazurok001.github.io/layout_stars/)
- [TEST REPORT LINK](https://Mazurok001.github.io/layout_stars/report/html_report/)

- [x] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [x] Each BEM block has its own separate file
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,75 @@
...
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(n + 5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```
Comment (DanilWeda) [inline src/style.css:75]: You can try use scss loop for this opetrations. 
Try it!

File: src/style.css [hunk 2]
```diff
@@ -1 +1,75 @@
...
+
+/* .container {
+   box-sizing: border-box;
+  width: 140px;
+  height: 160px;
+  padding: 22px;
+  display: flex;
+  flex-direction: column;
+} */
```
Comment (DanilWeda) [inline src/style.css:16]: remove commented if you don't need it

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Good job.
Check some small comments that i left!
Many thanks)

**Review by Mazurok001 — COMMENTED** [review]
remove commented

**Review by DanilWeda — APPROVED** [review]
Great! 
Approved! 
Thanks)

**Pull request closed without merging**